### PR TITLE
gemmi 0.4.9 (new formula)

### PR DESCRIPTION
**Pull request by blake-riley**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by sjackman** [comment]
```
==> brew linkage --test brewsci/bio/gemmi
==> FAILED
Unwanted system libraries:
  /lib/x86_64-linux-gnu/libz.so.1
```
https://github.com/brewsci/homebrew-bio/runs/4076248655?check_suite_focus=true#step:5:102

This error means that this formula ought to depend on `zlib`.
Because `zlib` is provided by macOS, you can add `uses_from_macos "zlib"`. You'll see examples of this style of dependency in other formulae.

File: Formula/gemmi.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/gemmi", "--help"
```
Comment (sjackman) [inline Formula/gemmi.rb:17]: You can use `assert_match` to ensure that `gemmi --help` has run correctly (didn't fail). You'll see examples of using `assert_match` and `shell_output` in other formulae.

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/gemmi.rb (hunk 1 above)
Comment (blake-riley) [reply Formula/gemmi.rb]: I'm now using assert_matches. Thanks for the pointer.

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Thank you for your first contribution to Brewsci/bio, Blake! 🥇

**Pull request merged**